app/registries/foreman/access_permissions.rb

```diff
@@ -351,6 +351,7 @@
     map.permission :ipmi_boot_hosts, { :hosts          => [:ipmi_boot],
                                        :"api/v2/hosts" => [:boot] }
     map.permission :forget_status_hosts, { :hosts => [:forget_status] }
+    map.permission :statuses_host, { :hosts => [:statuses] }
```

Comment:
This is not correct. Here you are creating new permission and in the controller, you are using the `view_hosts` permissions. If you intended to create permission, you'd need new line in controller and also add the perm in `db/seeds.d/020-permissions_list.rb`.
But I believe the `view_hosts` is the correct permission here and we should just add `:statuses` in allowed actions for that permission.